csharp/BUILDING.md

```diff
@@ -45,12 +44,6 @@ msbuild msbuild\ice.proj
 
 Upon completion, the Ice assemblies for .NET 8.0 are placed in the `lib\net8.0` folder.
 
-You can skip the build of the test suite with the `BuildDist` target:
-
-```shell
-msbuild msbuild\ice.proj /t:BuildDist
-```
```

Comment:
BuildDist was used before when we have separate targets for sources and tests.

Reply:
I'd rather remove BuildDist and not keep it as an alias for Build.

Reply:
Ok, I would in a follow up PR, need to fix CI builds that relay in all languages to provide a BuildDist target.